build/evergreen_config_lib/functions.py

```diff
@@ -543,5 +543,35 @@
         export IAM_AUTH_ECS_SECRET_ACCESS_KEY=${iam_auth_ecs_secret_access_key}
         sh ./.evergreen/run-aws-tests.sh ${TESTCASE}
         ''')
-    ))
+    )),
+    ('clone drivers-evergreen-tools', Function(
+        shell_exec(r'''
+        if [ ! -d "drivers-evergreen-tools" ]; then
+            git clone git@github.com:mongodb-labs/drivers-evergreen-tools.git --depth=1
+        fi
+        ''', test=False)
+    )),
+    ('start load balancer', Function(
+        shell_exec(r'''
+        export DRIVERS_TOOLS=./drivers-evergreen-tools
+        export MONGODB_URI="${MONGODB_URI}"
+        bash $DRIVERS_TOOLS/.evergreen/run-load-balancer.sh start
+        ''', test=False),
+        OD ([
+            ("command", "expansions.update"),
+            ("params", OD([("file", "lb-expansion.yml")]))
+        ]),
+    )),
+    ('stop load balancer', Function(
+        shell_exec(r'''
+        # Only run if a load balancer was started.
+        if [ -z "${SINGLE_MONGOS_LB_URI}" ]; then
```

Comment:
Is this going to be defined in the Evergreen web UI? I don't see any other references to it in this PR.

Reply:
This is updated by the `expansions.update` command in `start load balancer`. It adds both of the expansions: `SINGLE_MONGOS_LB_URI` and `MULTI_MONGOS_LB_URI`.